pkg/status/templates/forwarder.tmpl

```diff
@@ -9,7 +9,11 @@ Forwarder
   ============
   {{- range $key, $value := .Transactions }}
     {{- if and (ne $key "Errors") (ne $key "ErrorsByType") (ne $key "HTTPErrors") (ne $key "HTTPErrorsByCode") (ne $key "ConnectionEvents")}}
-    {{$key}}: {{humanize $value}}
+    {{$key}}
+    =================
+    {{- range $key, $value := $value }}
```

Comment:
What this change does?

Reply:
It updates the forwarder output to accommodate the changes to the expvar. Below is how it looks now:

```bash
  Transactions
  ============
    Bytes
    =================
      check_run_v1: 14,061
      intake: 3,711
      series_v1: 75,819

    ConnectionEvents
    =================
      ConnectSuccess: 134
      DNSSuccess: 0

    Count
    =================
      check_run_v1: 69
      intake: 5
      series_v1: 69

    Dropped
    =================
      check_run_v1: 5
      intake: 2
      series_v1: 7

    DroppedOnInput: 0

    OrchestratorCount
    =================

    Requeued
    =================
      check_run_v1: 56
      intake: 24
      series_v1: 19

    Retried
    =================
      check_run_v1: 12
      intake: 8
      series_v1: 7

    RetryQueueSize: 0

    Success
    =================
      check_run_v1: 64
      intake: 3
      series_v1: 62

    SuccessBytes
    =================
      check_run_v1: 13,051
      intake: 1,569
      series_v1: 68,569

    SuccessTotal: 129
```